src/fluree/db/query/subject_crawl/common.cljc

```diff
@@ -124,13 +124,16 @@
   - :order - :asc or :desc
   - :predicate - if type = :predicate, contains predicate pid or name
   - :variable - if type = :variable, contains variable name (not supported for simple subject crawl)"
-  [results {:keys [type order predicate]} limit]
+  [results {:keys [type order predicate]} limit offset]
   (if (= :variable type)
     (throw (ex-info "Ordering by a variable not supported in this type of query."
                     {:status 400 :error :db/invalid-query}))
     (let [sorted (cond-> (sort-by (fn [result] (get result predicate)) results)
                          (= :desc order) reverse)]
-      (vec (take limit sorted)))))
+      (into []
+            (comp (drop offset)
+                  (take limit))
```

Comment:
Boy these transducer comps always look backwards to me. But because it looks backwards, it must be right! ;)